### PR TITLE
Allow to exclude packages from deletion of test sub-package

### PR DESCRIPTION
**Pull request by EnyMan**
Some packages imports tests in their `__init__.py` files. Namely NetworkX.

To prevent their deletion and thus allow them to be importable. I added an option so the user can specify packages where `test` folder is not removed.

### CODE REVIEW

File: lambdipy/cli.py [hunk 1]
```diff
@@ -42,7 +42,8 @@ def version():
 @cli.command()
 @click.option('--from-pipenv', '-p', is_flag=True, help='Build dependencies from Pipfile.lock')
 @click.option('--include', '-i', multiple=True, help='Include these paths in the final build')
-def build(from_pipenv, include):
+@click.option('--exclude-tests', '-t', multiple=True, help='Exclude deletions of tests for these packages')
```
Comment (adikus) [inline lambdipy/cli.py]: Might be better to call this something like `keep-tests`. Or maybe `keep-tests-pattern`?

File: lambdipy/project_build.py [hunk 2]
```diff
@@ -240,6 +240,8 @@ def install_non_resolved_requirements(resolved_requirements, requirements, build
     if len(packages_to_install) > 0:
         print(f'Installing remaining packages via pip:{packages_to_install}')
 
+    exclude_tests_grep = '\|'.join(exclude_tests) if exclude_tests else '*'
```
Comment (adikus) [inline lambdipy/project_build.py]: What do you think about `exclude_tests_pattern`?

**Review by adikus — APPROVED** [review]
Thanks! Looks good.
Can you also bump the version in https://github.com/customink/lambdipy/blob/598d78f0575f9719a0efa0feebb009be8cd364e6/lambdipy/__init__.py?

File: lambdipy/project_build.py (hunk 2 above)
Comment (EnyMan) [reply lambdipy/project_build.py]: much better

File: lambdipy/cli.py (hunk 1 above)
Comment (EnyMan) [reply lambdipy/cli.py]: I like `keep-tests`

**Review by adikus — APPROVED** [review]

**Pull request merged**